impala/tests/conftest.py

```diff
@@ -113,3 +113,9 @@ def cur(con):
     cur = con.cursor()
     yield cur
     cur.close()
+
+@fixture(scope='session')
+def cur_noconv(con):
+    cur = con.cursor(convert_types=True, convert_strings_to_unicode=False)
```

Comment:
here and in test_data_types.py: is there a reason for removing the final new lines?

Reply:
It's by mistake, fixed